### ISSUE TITLE
UI Fix - Events section

### ISSUE THREAD

**Issue by johnathantng** [inland-empire-software-development/landing#5]
The events section is a bit cluttered. 

- I think a fix for it would be to take out the two rows, because it'll be less confusing that way. Especially since there's no borders that indicate how you should read it.

- Maybe have some visual cues for how the user should read the events (borders, images, icons, etc.) Anything to indicate the flow/direction for how the user should read it. 

- The title the events should be emphasized and maybe even a short summary or description of each event. The event name should be "visually loud" so that it'll be the first thing that catches the user's eyes. The description(?), date, location, time should come after.

**Comment by lloan** [comment]
I agree. Since we're using Bootstrap, we should stick to the components provided by BootStrap. 

Here is a link to the `card` component -> https://getbootstrap.com/docs/4.3/components/card/ 

We could use this for each event that is pulled from the Meetup.com API. 

Building off your first point, we could have a horizontal carousel, slides from right to left, showing the latest event first. **OR** we could have the first row be the latest events and have a -show more- link below that row, centered. Once the user clicks it, another row is 'lazy loaded' - providing the user with more information about past events.
